views/_includes/banner.pug

```diff
@@ -1,6 +1,6 @@
-mixin banner(variant, spanChar)
+mixin banner(variant, spanChar='i')
```

Comment:
Yeah, seems better.

Reply:
Actually, so far we're only passing in `i` or nothing, so we could just not have a second parameter and write an 'if' based on the string that gets passed in.

But either works, really.